models/zeroex/arbitrum/zeroex_arbitrum_api_fills.sql

```diff
@@ -309,7 +309,8 @@ SELECT
              ELSE COALESCE((all_tx.maker_token_amount_raw / pow(10, mp.decimals)) * mp.price, (all_tx.taker_token_amount_raw / pow(10, tp.decimals)) * tp.price)
              END AS volume_usd,
         tx.from AS tx_from,
-        tx.to AS tx_to
+        tx.to AS tx_to,
+         'arbitrum' AS blockchain
```

Comment:
minor point, but can you add `blockchain` column to each blockchain schema file?
i.e. arbitrum here:
https://github.com/duneanalytics/spellbook/blob/main/models/zeroex/arbitrum/zeroex_arbitrum_schema.yml

keep in mind, the deduped spell has the column already below it and uses `&` jinja syntax, meaning that should likely be moved to `api_fills` section above, so then you can use `*` syntax in the second spell.

hope that makes sense!